### ISSUE TITLE
file.replace('.json', '') crashes application on startup when custom preset is present

### ISSUE THREAD

**Issue by avanishsubbiah** [ArtyIF/AdwCustomizer#15]
When loading a custom preset on startup the application crashes because it attempts to apply the `.replace()` method on a file pointer instead of the file name itself. This is the error generated on startup:

```
Traceback (most recent call last):
  File "/app/share/adwcustomizer/adwcustomizer/main.py", line 141, in do_activate
    self.custom_presets[file.replace('.json', '')] = preset['name']
AttributeError: '_io.TextIOWrapper' object has no attribute 'replace'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/share/adwcustomizer/adwcustomizer/main.py", line 144, in do_activate
    "error": "Failed to load preset: " + file,
TypeError: can only concatenate str (not "_io.TextIOWrapper") to str
```

This is due to line 140 in `main.py` and the error is resolved by changing the line from:
```
self.custom_presets[file.replace('.json', '')] = preset['name']
```
to,
```
self.custom_presets[os.path.basename(file.name).replace('.json', '')] = preset['name']
```

By making this change `main.py` successfully gets the filename of the custom preset in the `presets` folder as a string and loads the preset on startup correctly.

**Comment by ArtyIF** [comment]
Whoops, that was an oversight when I was renaming variables.

**Comment by ArtyIF** [comment]
Fixed in d2cdd0c29979e2159db81d9cee71223445ac5f28
